images/win/scripts/Installers/Install-JavaTools.ps1

```diff
@@ -87,19 +96,40 @@ function Install-JavaFromAdoptOpenJDK {
     New-Item -ItemType File -Path $javaVersionPath -Name "$Architecture.complete" | Out-Null
 }
 
-$jdkVersions = (Get-ToolsetContent).java.versions
+$jdkVendors = (Get-ToolsetContent).java.vendors
```

Comment:
nitpick: we can create an additional variable for  `(Get-ToolsetContent).java` part, like:
```
$toolsetJava = (Get-ToolsetContent).java
$jdkVendors = $toolsetJava.vendors
```